CHANGELOG/CHANGELOG-2.5.md

```diff
@@ -0,0 +1,98 @@
+## Changes by Kind
```

Comment:
Oops I missed this. Can you also add the headers with the release number?

Reply:
sure, I didn't notice that, thanks